boxwise_flask/type_defs.py

```diff
@@ -10,6 +10,9 @@
         allUsers: [User]
         user(email: String): User
     }
+    type Mutation {
```

Comment:
Because this page is going to get pretty large. I started splitting the mutations and queries and type defs into seperate strings. Maybe they should be seperate files too. you can import them and join them all by wrapping gql when making the schema